CHANGELOG.md

```diff
@@ -17,11 +17,13 @@
 ## Bug Fixes
 
 ## Internal
+Upgraded to OkHttp 3.9.0 .
+Upgraded to RxJava 2.1.4 .
 
 ## Credits
 
 
-## 4.0.0-BETA3 (YYYY-MM-DD)
+## 4.0.0-BETA3 (2017-09-18)
```

Comment:
This doesn't look right. We haven't released a BETA3 publically.

Reply:
@cmelchior I can find the BETA3 Published here http://oss.jfrog.org/oss-snapshot-local/io/realm/realm-gradle-plugin/ and http://oss.jfrog.org/oss-snapshot-local/io/realm/realm-gradle-plugin/4.0.0-BETA3-SNAPSHOT/  I am also using it in my project,

Reply:
Yes, but that is a `-SNAPSHOT` release. They are done automatically from `master` and is not considered a "real" release. Only releases done from the `releases` branch are. They are also tagged: https://github.com/realm/realm-java/releases

Reply:
Yes, thats why its appended with 4.0.0-BETA3 , "BETA3", real release wont have -BETA Tag, I have followed the similar convention as per the change log since it was done for BETA1 and BETA2 in the same way. See https://github.com/vivekkiran/realm-java/blob/1bc72f8ce0136521270928347707adabf0fef055/CHANGELOG.md , in that case we shouldn't be logging any -BETA Releases in the Changelog.

Reply:
The BETA releases were mostly private releases done from the `master-4.0` branch. They should have been collapsed when moving to `master`, and they will be in the next few days. I do realize this is confusing. Sorry about that.